🐞 Sample data generation failed

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I downloaded and built the code on MacOS 12.4 today.  The output of the build process showed no errors.  I attempted to build a sample using make.  Here is what happened.

```
$ make SAMPLE_BALLOT_COUNT=100 generate-sample-data
🔁 GENERATE Sample Data
poetry run python3 src/electionguard_tools/scripts/sample_generator.py -m "hamilton-general" -n 100 -s 50
/usr/local/lib/python3.9/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
  warnings.warn(
[52500:2022-06-06 13:46:16,850]:ERROR:elgamal.py.decrypt:#L143: MAC verification failed in decryption.
Traceback (most recent call last):
  File "/Users/ramsdell/rep/electionguard-python/src/electionguard_tools/scripts/sample_generator.py", line 249, in <module>
    ElectionSampleDataGenerator().generate(
  File "/Users/ramsdell/rep/electionguard-python/src/electionguard_tools/scripts/sample_generator.py", line 83, in generate
    ) = self.election_factory.get_sample_manifest_with_encryption_context(
  File "/Users/ramsdell/rep/electionguard-python/src/electionguard_tools/factories/election_factory.py", line 128, in get_sample_manifest_with_encryption_context
    KeyCeremonyOrchestrator.perform_full_ceremony(guardians, mediator)
  File "/Users/ramsdell/rep/electionguard-python/src/electionguard_tools/helpers/key_ceremony_orchestrator.py", line 29, in perform_full_ceremony
    KeyCeremonyOrchestrator.perform_round_3(guardians, mediator)
  File "/Users/ramsdell/rep/electionguard-python/src/electionguard_tools/helpers/key_ceremony_orchestrator.py", line 69, in perform_round_3
    guardian.verify_election_partial_key_backup(
  File "/Users/ramsdell/rep/electionguard-python/src/electionguard/guardian.py", line 368, in verify_election_partial_key_backup
    return verify_election_partial_key_backup(
  File "/Users/ramsdell/rep/electionguard-python/src/electionguard/key_ceremony.py", line 266, in verify_election_partial_key_backup
    get_optional(bytes_optional)
  File "/Users/ramsdell/rep/electionguard-python/src/electionguard/utils.py", line 72, in get_optional
    assert optional is not None, "Unwrap called on None"
AssertionError: Unwrap called on None
make: *** [generate-sample-data] Error 1
$ 
```

### Expected Behavior

Sample data should be generated.

### Steps To Reproduce

_No response_

### Environment

```markdown
- OS: MacOS 12.4.
```


### Anything else?

_No response_

--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,4 @@
-.PHONY: all environment openssl-fix install install-gmp install-gmp-mac install-gmp-linux install-gmp-windows install-mkdocs auto-lint validate test test-example bench coverage coverage-html coverage-xml coverage-erase fetch-sample-data generate-sample-data
+.PHONY: all environment openssl-fix install install-gmp install-gmp-mac install-gmp-linux install-gmp-windows install-mkdocs auto-lint validate test test-example bench coverage coverage-html coverage-xml coverage-erase fetch-sample-data
 
 CODE_COVERAGE ?= 90
 OS ?= $(shell python3 -c 'import platform; print(platform.system())')
@@ -186,10 +186,6 @@ endif
 	unzip -o sample-data.zip
 	unzip sample-data.zip
 
-generate-sample-data:
-	@echo 🔁 GENERATE Sample Data
-	poetry run python3 src/electionguard_tools/scripts/sample_generator.py -m "hamilton-general" -n $(SAMPLE_BALLOT_COUNT) -s $(SAMPLE_BALLOT_SPOIL_RATE)
-
 # Publish
 publish:
 	poetry publish
@@ -205,12 +201,12 @@ publish-test-ci:
 	@echo 🚀 PUBLISH TEST
 	poetry publish --repository testpypi --username __token__ --password $(TEST_PYPI_TOKEN)
 
-publish-validate:	
+publish-validate:
 	@echo ✅ VALIDATE
 	python3 -m pip install --no-deps electionguard
 	python3 -c 'import electionguard'
 
-publish-validate-test:	
+publish-validate-test:
 	@echo ✅ VALIDATE
 	python3 -m pip install --index-url https://test.pypi.org/simple/ --no-deps electionguard
 	python3 -c 'import electionguard'

